builders/get-started/eth-compare/consensus-finality.md

```diff
@@ -42,61 +42,81 @@ However, the deterministic finality of Moonbeam can be used to provide a better
 
 The following sections outline how you can check for transaction finality using both the Ethereum JSON-RPC (custom Web3 request) and the Substrate (Polkadot) JSON-RPC.
 
-## Checking Tx Finality with Ethereum Libraries
-
-You can make calls to the Substrate JSON-RPC using the `send` method of both [Web3.js](https://web3js.readthedocs.io/) and [Ethers.js](https://docs.ethers.io/).
-
-Custom RPC requests are also possible using [Web3.py](https://web3py.readthedocs.io/) with the `make_request` method. You can use the Web3.js example as a baseline.
+## Checking Tx Finality with Moonbeam RPC Endpoints {: #checking-tx-finality-with-moonbeam-rpc-endpoints }
+
+Moonbeam has added support for two new RPC endpoints, `moon_isBlockFinalized` and `moon_isTxFinalized` to the Moonbeam node, that are useful for checking whether an on-chain event is finalized. The information on these two endpoints are as follows:
+
+=== "moon_isBlockFinalized"
+    |   Variable   |                                      Value                                       |
+    |:------------:|:--------------------------------------------------------------------------------|
+    |   Endpoint |                        `moon_isBlockFinalized`                     |
+    |   Description   | Check for the finality of the block given by its block hash. |
+    |  Parameters |    `block_hash`: **STRING** The hash of the block, accepts either Substrate-style or ethereum-style block hash as its input.                     | 
+    |  Returns | `result`: **BOOLEAN** Returns `True` if the block is finalized, `False` if the block is not finalized or not found.  | 
```

Comment:
the parameters and returns rows look a little bit weird

also are `True` and `False` supposed to be capitalized?

and lastly, I would remove the period at the end of all of the sentences in tables

Reply:
Removed the periods, and changed true and false to lower case to match with the results. Thanks.